wgpu-hal/src/gles/queue.rs

```diff
@@ -533,7 +533,7 @@ impl super::Queue {
                         0,
                     );
                 }
-                gl.color_mask(true, true, true, true);
+                gl.color_mask(true, true, true, false);
```

Comment:
ok, this is certainly not the fix we want.
Isn't the shadow example outputting all alpha = 1 anyway?

This sounds like one of the following possibilities:
  - alpha compositing mode is messed up, and we produce alpha that isn't 1.0
  - driver bug